.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+version: 2
+
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: circleci/node:8.11
+
+jobs:
+  publish-version:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          - v1-dependencies-
+
+      - attach_workspace:
+          at: ~/repo
+      - run:
+          name: Authenticate with registry
+          command: echo "//registry.npmjs.org/:_authToken=$npm_TOKEN" > ~/repo/.npmrc
+
+      - run: yarn version --new-version $CIRCLE_BRANCH
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "package.json" }}
+
+workflows:
+  version: 2
+  publish:
+    jobs:
+      - publish-version:
+          filters:
+            branches:
+              only: /^v.*/
```

Comment:
shouldn't we follow git flow naming for release branches and format them like this: `release/vx.x.x` ?